README.Rmd

```diff
@@ -33,11 +33,10 @@ Tools to enable flexible and efficient hierarchical nowcasting of right-truncate
 ## Getting started and learning more
 
 This README is a good place to get started with `epinowcast`, in particular the following installation and quick start sections. As you make use of the package, or if your problem requires a richer feature set than presented here, we also provide a range of other documentation, case studies, and spaces for the community to interact with each other. Below is a short list of current resources.
-
-- [Package website](https://package.epinowcast.org/): This includes a function reference, model outline, and case studies making use of the package. The development version of our documentation (corresponding to our `develop` branch) is available [here](https://package.epinowcast.org/dev/).
+- [Package website](https://package.epinowcast.org/): This includes a function reference, model outline, and case studies making use of the package. This site refers to the release version of our package which can be installed from our Universe or from the latest GitHub release (see installation instructions). The development version of our documentation (corresponding to our `main` branch on GitHub) is available [here](https://package.epinowcast.org/dev/).
 - [Organisation website](https://www.epinowcast.org/): This includes links to our other resources as well as guest posts from community members and schedules for any related seminars being run by community members.
 - [Directory of example scripts](https://github.com/epinowcast/epinowcast/tree/main/inst/examples): Not as fleshed out as our complete case studies these scripts are used during package development and each showcase a subset of package functionality. Often newly introduced features will be explored here before surfacing in other areas of our documentation.
-- [Community forum](https://community.epinowcast.org/): Our community forum is where development of tools is discussed, along with related research from our members and discussions between users. If you are interested in real-time analysis of infectious disease this is likely a good place to start regardless of if you end up making use of `epinowcast`.
+- [Community forum](https://community.epinowcast.org/): Our community forum is where development of methods and tools is discussed, along with related research from our members and discussions between users. If you are interested in real-time analysis of infectious disease this is likely a good place to start regardless of if you end up making use of `epinowcast`.
```

Comment:
What's the plan to for use of the github discussion board? I'm generally pro one-job-one-tool -- does it make sense to transition that board to something here? Or are there specific capabilities over that not on offer here?

Reply:
I thought we might want to use it for questions about how to use the package but not really sure.